src/config.js

```diff
@@ -7,6 +7,7 @@ const dirname = path.dirname(fileURLToPath(import.meta.url))
 const isProduction = process.env.NODE_ENV === 'production'
```

Comment:
Moving `index.js` out of `config` folder as feels redundant having a folder with only an index file.

`config.js` feels like it cognitively equates to `config/index.js`